app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :require_admin
```

Comment:
Probably wouldn't put this helper method declaration in the `ApplicationController` if it's not defined there--even as a no-op for overriding.